### PR TITLE
Rearrange things and reformat to flake8 standards.

### PR DESCRIPTION
**Pull request by gwax**

### CODE REVIEW

File: temporal_sqlalchemy/nine.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
-# THANKS PYTHON
+"""Python compatibility helpers."""
```
Comment (multigl) [inline temporal_sqlalchemy/nine.py]: lol

File: tests/models.py [hunk 2]
```diff
@@ -143,9 +155,12 @@ class SimpleTable(RelatedBase):
     rel = orm.relationship(RelatedTable, info={'temporal_on': 'rel_id'})
 
 
-@temporal_sqlalchemy.add_clock('really_really_really_really_really_long_column')
+REALLY_REALLY = 'really_' * 5
```
Comment (multigl) [inline tests/models.py]: nice

File: circle.yml [hunk 3]
```diff
@@ -12,3 +12,4 @@ dependencies:
 test:
   override:
     - py.test -r w tests --junitxml=$CIRCLE_TEST_REPORTS/temporal-sqlalchemy.xml
+    - flake8
```
Comment (multigl) [inline circle.yml]: does flake8 default to 80 char lines? I'm comfortable with 80-120

Comment (gwax) [inline circle.yml]: flake8 defaults to <80, which is to say max 79.

This conforms with PEP8.

The general rule of thumb I tend to hear is: open source should stick to 80 characters but private projects might want to raise the limit to 100 or 120, depending on team preferences.

Personally, I am inclined to advocate for strict PEP8 adherence for open source projects.

Comment (multigl) [inline circle.yml]: it's easier to check for, I'm okay with it

Comment (multigl) [inline circle.yml]: PEP8 that is

**Pull request merged**